packages/builder/src/nav/BackendRightNav.svelte

```diff
@@ -0,0 +1,142 @@
+<script>
+  import { store } from "../builderStore"
+  import HierarchyRow from "./HierarchyRow.svelte"
+  import { hierarchy as hierarchyFunctions } from "../../../core/src"
+  import NavItem from "./NavItem.svelte"
+
+  var acc = document.getElementsByClassName("accordion")
+  var i
+
+  for (i = 0; i < acc.length; i++) {
```

Comment:
We want to do this with svelte state and keep it inside the framework rather than just vanilla JS DOM manipulation